app/src/main/res/values/strings.xml

```diff
@@ -1046,7 +1046,7 @@
     <string name="checking_server">checking server</string>
     <string name="below_min_apk">app is below allowed version. please update the app to the latest version.</string>
     <string name="add_note">add a note (optional)</string>
-    <string name="visit_count">%s (%d %s)</string>
+    <string name="visit_count" translatable="false">%s (%d %s)</string>
```

Comment:
adding `translatable="false"` means there is a language that the string is not translated in. in this case it is the somali language